### DISCUSSION TITLE
How to get Web URL with Bing Search in Azure Chat Completion

### DISCUSSION THREAD

**Discussion by KworkLan** [microsoft/semantic-kernel#4415] [General]
Do we have any example to let me get Web URL with Bing search plugin? 
Thank you

**Comment by vasireddy** [comment]
> Do we have any example to let me get Web URL with Bing search plugin? 
> 
> Thank you

Please review below classes. hope it helps

dotnet/samples/KernelSyntaxExamples/Example07_BingAndGooglePlugins.cs

dotnet/samples/KernelSyntaxExamples/Example11_WebSearchQueries.cs

**Comment by KworkLan** [comment]
Yes, I look at them but it just result text content, I could not get URL result like this :
![image](https://github.com/microsoft/semantic-kernel/assets/114917455/6f7ff7fd-2add-4adf-81db-4a94320194f8)
I would like to get search URL

**Reply by vasireddy** [reply]
Thank you for clarifying. I have reviewed "BingConnector" and seems like it is only considering "snippet" property. 

![image](https://github.com/microsoft/semantic-kernel/assets/2259161/a0145cc6-873c-4836-89e1-c128ba46c235)

however, "BingSearchResponse" has "url" property.

![image](https://github.com/microsoft/semantic-kernel/assets/2259161/4dfcd9fa-ec12-4d9a-ab6a-9a903a04a6e8)


May be enhance kernel plugin or build your own ?
